quartz/components/scripts/util.ts

```diff
@@ -43,3 +43,20 @@ export async function fetchCanonical(url: URL): Promise<Response> {
   const [_, redirect] = text.match(canonicalRegex) ?? []
   return redirect ? fetch(`${new URL(redirect, url)}`) : res
 }
+
+export function assignActiveClassToDrawerButton(isactive: boolean) {
```

Comment:
Add the Function Document here as it is in the `util` folder.
```suggestion
/**
* Documentation
*/
export function assignActiveClassToDrawerButton(isactive: boolean) {
```